### ISSUE TITLE
Android 14 - Bitwarden autofill cannot be enabled.

### ISSUE THREAD

**Issue by yurividal** [bitwarden/mobile#2826]
### Steps To Reproduce

Open bitwarden, go to settings, auto-fill service. Try to enable auto-fill service. The button goes back to off.

If you go in the Amdroid security settings, Bitwarden doesn't have the option to enable autofill, while 1password and dashlane have the option.

### Expected Result

Bitwarden should allow turning on autofill service.

### Actual Result

Unable to use native android autofill service with bitwarden on Android 14.

### Screenshots or Videos

![IMG-20231015-WA0014](https://github.com/bitwarden/mobile/assets/8555749/6b064078-f5c9-4021-8e8c-2dc8e14e2d4d)
![Screenshot_20231015_141743_Settings](https://github.com/bitwarden/mobile/assets/8555749/bbf2edcd-765f-41c6-994f-4ffaf13bc352)

### Additional Context

Samsung Galaxy S23 running Android 14 (OneUI 6)

### Operating System

Android

### Operating System Version

14

### Device

Galaxy S23

### Build Version

Version: 2023.9.2 (8002)

### Beta

- [ ] Using a pre-release version of the application.

**Comment by SergeantConfused** [comment]
Hi @yurividal,

Thank you for your report. Are you using a Beta release of Android 14 for Samsung (One UI)?


Thank you in advance,

**Comment by yurividal** [comment]
> Hi @yurividal,
> 
> Thank you for your report. Are you using a Beta release of Android 14 for Samsung (One UI)?
> 
> Thank you in advance,

Yes I am. Initially I thought it could be a problem with OneUI 6 (beta), but after seeing that dashlane and 1password work fine and show the toggle, it made me realize it might actually be something with the bitwarden app

**Comment by yurividal** [comment]
@SergeantConfused samsung has acknowledged that Bitwarden will need to be updated in order to work with OneUI6.

![Screenshot_20231017_050127_Samsung Members](https://github.com/bitwarden/mobile/assets/8555749/eaf02494-192d-4f16-9c23-fd63f949e4d0)

**Comment by yurividal** [comment]
It seems that with the latest update that Samsung dropped yesterday, this has been sort of fixed.
The only thing is that you still can't enable the service from inside of bitwarden. You need to go into the system settings and enable it from there.

Details here: https://www.reddit.com/r/oneui/comments/17goag9/beta_9_actually_fixes_thirdparty_autofill_in/

**Comment by HelloKS** [comment]
I also can confirm this problem is fixed in beta 9 (ZWJM). You may need to enable "Appear on top" permission for autofill popup.
![Screenshot_20231026_234956_Settings](https://github.com/bitwarden/mobile/assets/3853894/6bde702f-c8c5-40d9-9cbb-87c5ce985bc8)
![Screenshot_20231026_235038_Brave](https://github.com/bitwarden/mobile/assets/3853894/69d5f7da-d719-4797-87a2-a732a9248377)

**Comment by yurividal** [comment]
Yeah, its definitely not 100% fixed, as its still impossible to turn on using the toggle inside bitwarden. But, at least it works now, whem you enable it via the system settings..
